pkg/graph/schema.resolvers.go

```diff
@@ -1949,14 +1964,32 @@ func (r *systemIntakeDocumentResolver) CanView(ctx context.Context, obj *models.
 	return resolvers.CanViewDocument(ctx, grbUsers, obj), nil
 }
 
+// VotingRole is the resolver for the votingRole field.
+func (r *systemIntakeGRBReviewDiscussionPostResolver) VotingRole(ctx context.Context, obj *models.SystemIntakeGRBReviewDiscussionPost) (*models.SystemIntakeGRBReviewerVotingRole, error) {
+	if obj.VotingRole == nil {
+		return nil, nil
+	}
+	strVal := *obj.VotingRole
+	return helpers.PointerTo(models.SystemIntakeGRBReviewerVotingRole(strVal)), nil
```

Comment:
would it be worthwhile to match the naming to get rid of this resolver? it's possible i am overlooking something, this looks like ping ponging between similar types

Reply:
Yeah, I think we're just using 2 different types that have the same definition.

@samoddball you wanna throw a PR up with those changes to nuke these resolvers?